cmd/cluster/kubevirt/create.go

```diff
@@ -30,15 +32,18 @@ func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
 		Cores:                     2,
 		ContainerDiskImage:        "",
 		RootVolumeSize:            16,
+		ExtClusterKubeConfigFile:  "",
 	}
 
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.APIServerAddress, "api-server-address", opts.KubevirtPlatform.APIServerAddress, "The API server address that should be used for components outside the control plane")
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.Memory, "memory", opts.KubevirtPlatform.Memory, "The amount of memory which is visible inside the Guest OS (type BinarySI, e.g. 5Gi, 100Mi)")
-	cmd.Flags().Uint32Var(&opts.KubevirtPlatform.Cores, "cores", opts.KubevirtPlatform.Cores, "The number of cores inside the vmi, Must be a value greater or equal 1")
+	cmd.Flags().Uint32Var(&opts.KubevirtPlatform.Cores, "cores", opts.KubevirtPlatform.Cores, "The number of cores inside the vmi, Must be a value greater than or equal to 1")
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.RootVolumeStorageClass, "root-volume-storage-class", opts.KubevirtPlatform.RootVolumeStorageClass, "The storage class to use for machines in the NodePool")
 	cmd.Flags().Uint32Var(&opts.KubevirtPlatform.RootVolumeSize, "root-volume-size", opts.KubevirtPlatform.RootVolumeSize, "The size of the root volume for machines in the NodePool in Gi")
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.ContainerDiskImage, "containerdisk", opts.KubevirtPlatform.ContainerDiskImage, "A reference to docker image with the embedded disk to be used to create the machines")
 	cmd.Flags().StringVar(&opts.KubevirtPlatform.ServicePublishingStrategy, "service-publishing-strategy", opts.KubevirtPlatform.ServicePublishingStrategy, fmt.Sprintf("Define how to expose the cluster services. Supported options: %s (Use LoadBalancer and Route to expose services), %s (Select a random node to expose service access through)", IngressServicePublishingStrategy, NodePortServicePublishingStrategy))
+	cmd.Flags().StringVar(&opts.KubevirtPlatform.ExtClusterKubeConfigFile, "external-cluster-kubeconfig-file", opts.KubevirtPlatform.ExtClusterKubeConfigFile, "Path to a kubeconfig file of an external cluster to be used to create the guest clusters nodes onto")
```

Comment:
We'll probably need to discuss these terms with the hypershift team a bit. it's possible that `external-cluster-kubeconfig-file` will need to turn into something like `infra-cluster-kubeconfig-file`. Somehow we need to converge our terminology here to match both the user facing API and our internal representation of this kubeconfig.

Reply:
updated as suggested.